shimmingtoolbox/coils/coil.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+
+class Coil(object):
+    """
+    Coil profile object that stores coil profiles and there constraints
+
+    Attributes:
+        x (int): Amount of pixels in the X direction
+        y (int): Amount of pixels in the Y direction
+        z (int): Amount of pixels in the Z direction
+        n (int): Amount of channels in the coil profile
+        profiles (numpy.ndarray): (X, Y, Z, N) 4d array of N 3d coil profiles
+        coef_sum_max (float): Contains the maximum value for the sum of the coefficients
```

Comment:
we might want to reconsider the naming of the variable, given that the ACDC coil constraint unit is "amp"

Reply:
I agree that `coef` might not be the best name however the `Coil` class does not only support "amps" as a unit. We can use it with spherical harmonics which don't have "amps" as units. The `coef` really only tells us how many times we can multiply the `profile`, regardless of the unit. 

So from a design perspective I feel like we are restricting the use case by saying amps but from a user's perspective it gives a lot more intuition for what I think will be the scenario used by people most of the time.